.github/workflows/android.yml

```diff
@@ -28,6 +28,10 @@ jobs:
         cache: gradle
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
+    - name: Load Google Service file
```

Comment:
i added a github secret for running android CI (see comment in .gitignore for doing this locally)